Testreglar/1.4.3/Nett/nett-1.4.3a.json

```diff
@@ -0,0 +1,253 @@
+{
+	"namn": "Nett-1.4.3a Det er tilstrekkelig kontrast mellom tekst og bakgrunn 2023",
+	"id": "nett-1.4.3a",
+	"testlabId": 439,
+	"versjon": "1.0",
+	"type": "Nett",
+	"spraak": "nb",
+	"kravTilSamsvar": "<p>Tekst</p>\r\n<ul>\r\n<li>Tekst har kontrast på minst 4,5:1 mot bakgrunnen</li>\r\n<li>Stor tekst har kontrast på minst 3,0:1 mot bakgrunnen</li>\r\n</ul>",
```

Comment:
Må ha en sammenheng:  For eksempel:  For nettsider med tekst gjelder følgende: ...